src/xenia/kernel/kernel_state.h

```diff
@@ -153,13 +153,10 @@ struct DPCImpersonationScope {
 struct KernelVersion {
   union {
     xe::be<uint64_t> value;
-
```

Comment:
This part now is incorrect. It was written that way, because we want either to write whole value as a 8bytes long or get specific parts of it

Reply:
Yeah, I had build failures around this before, but fixing this back now compiles fine? Maybe it was a critical warning from the compiler before I remember to disable them again. I'll change this back shortly since it works now XD

Reply:
It's back as it was before now :)